R/activate.R

```diff
@@ -149,16 +149,15 @@ renv_activate_prompt <- function(action, library, prompt, project) {
   fmt <- lines(
     "",
     "This project has not yet been activated.",
-    "Activating this project will ensure the project library is used when %s() is called.",
-    "Please see `?renv::activate` for more details.",
+    "This means that %s() will modify your global library.",
```

Comment:
This line would only be true if this were called in response to restore / clean; if we kept the snapshot prompt then it wouldn't be accurate (so I guess this PR depends on https://github.com/rstudio/renv/pull/1222).